sdk/face/ai-vision-face-rest/test/public/faceSessionClient.spec.ts

```diff
@@ -0,0 +1,463 @@
+  });
+
+  it.runIf(isLiveMode())('TestCreateVerifySessionWithImage', async () => {
+    const deviceCorrelationId = recorder.variable('deviceCorrelationId', randomUUID());
+    const createLivenessSessionResponse = await client.path('/detectLivenessWithVerify/singleModal/sessions').post({
+      contentType: 'multipart/form-data',
+      body: [
+        new CreateLivenessWithVerifySessionContentVerifyImagePartDescriptor({
+            name: 'VerifyImage',
+            body: readFileSync('samples-dev/data/detection1.jpg'),
+        }),
+        new CreateLivenessWithVerifySessionContentParametersPartDescriptor({
+            name: 'Parameters',
+            body: {
+                livenessOperationMode: 'Passive',
+                sendResultsToClient: false,
+                authTokenTimeToLiveInSeconds: 60,
+                deviceCorrelationId: deviceCorrelationId,
+            },
+        }),
+      ],
+    }) as CreateLivenessWithVerifySessionWithVerifyImage200Response;
+    assert.equal(createLivenessSessionResponse.status, '200');
+    assert.isNotEmpty(createLivenessSessionResponse.body.sessionId);
+    assert.isNotEmpty(createLivenessSessionResponse.body.authToken);
+    assert.isNotEmpty(createLivenessSessionResponse.body.verifyImage?.faceRectangle);
+    assert.isNotEmpty(createLivenessSessionResponse.body.verifyImage?.qualityForRecognition);
+
+    const { sessionId } = createLivenessSessionResponse.body;
+
+    const getLivenessSessionResultResponse = await client.path('/detectLivenessWithVerify/singleModal/sessions/{sessionId}', sessionId).get() as GetLivenessWithVerifySessionResult200Response;
+    assert.equal(getLivenessSessionResultResponse.status, '200');
+    assert.equal(getLivenessSessionResultResponse.body.deviceCorrelationId, deviceCorrelationId);
+
+    const deleteLivenessSessionResponse = await client.path('/detectLivenessWithVerify/singleModal/sessions/{sessionId}', sessionId).delete() as DeleteLivenessWithVerifySession200Response;
+    assert.equal(deleteLivenessSessionResponse.status, '200');
+  });
+
+  it('TestListVerifySessions', async () => {
+    const createSession = async () => {
+      const deviceCorrelationId = recorder.variable('deviceCorrelationId', randomUUID());
+      const createLivenessSessionResponse = await client.path('/detectLivenessWithVerify/singleModal/sessions').post({
+        body: {
+          livenessOperationMode: 'Passive',
+          deviceCorrelationId,
+        },
+      }) as CreateLivenessWithVerifySession200Response;
+      assert.equal(createLivenessSessionResponse.status, '200');
+      return [createLivenessSessionResponse.body.sessionId, deviceCorrelationId];
+    };
+
+    const sessions = Object.fromEntries(await Promise.all([createSession(), createSession()]));
+    console.log(sessions);
+
+    const getLivenessSessionsResponse = await client.path('/detectLivenessWithVerify/singleModal/sessions').get() as GetLivenessWithVerifySessions200Response;
```

Comment:
Maybe also be sure to remove all unnecessary casting ?

Reply:
The castings are necessary because we are reading their fields.